src/datasources/network/fetch.network.service.spec.ts

```diff
@@ -234,27 +252,15 @@ describe('AxiosNetworkService', () => {
   });
 
   describe('DELETE requests', () => {
-    it(`delete calls axios delete`, async () => {
-      const url = faker.internet.url({ appendSlash: false });
-      const data = { some_data: 'some_data' };
-
-      await target.delete(url, data);
-
-      expect(axiosMock.delete).toHaveBeenCalledTimes(1);
-      expect(axiosMock.delete).toHaveBeenCalledWith(url, { data: data });
-    });
-
-    it(`delete forwards unknown error as NetworkOtherError`, async () => {
+    it(`get uses DELETE method`, async () => {
```

Comment:
"delete uses DELETE method"

Reply:
Fixed in c9ba85577a8d93c61ebf5d8aa38d80fbcaf2aebe.